### PR TITLE
Create Pigeonhole_Sort.js

### LINKED ISSUE #2149: Pigeonhole Sort
- [x] C
- [x] C++
- [x] C# - @Harikrishnan6336
- [ ] CoffeeScript
- [ ] Dart - @heisastark
- [x] Go - @m-code12
- [x] Java
- [ ] JS - @sheetalssr
- [ ] Kotlin - @Anushka118
- [ ] Php
- [x] Python
- [x] Readme
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by sheetalssr**
Fixes #2149 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- pigeonhole sort in javascript
- 

#### Languages Used:
- javascript
- 

#### File Added:
- Pigeonhole_Sort.js 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by sheetalssr** [comment]
done with the indentation thing. kindly check.

**Comment by sheetalssr** [comment]
Kindly review this as early as possible , so that I can move to the next issue.

File: Pigeonhole_Sort/Pigeonhole_Sort.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//taking dynamic input array to be sorted
+var i;
+var n=prompt("enter the size of your array");
+var arr = [];
```
Comment (asha15) [inline Pigeonhole_Sort/Pigeonhole_Sort.js]: keep a line space after this

File: Pigeonhole_Sort/Pigeonhole_Sort.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+for(i=0; i<n; i++) 
+{
+    //Taking Input from user
+    arr[i] = prompt('Enter Element ' + (i+1));
+}
```
Comment (asha15) [inline Pigeonhole_Sort/Pigeonhole_Sort.js]: keep a line space after this

**Pull request closed without merging**